### ISSUE TITLE
Training loss value is very large

### ISSUE THREAD

**Issue by gzYou** [Kulbear/ivus-segmentation-icsm2018#3]
Hi，
I got your code and ran it. But the training loss value is always very large and it keeps 190 000 levels.

**Comment by Kulbear** [comment]
Hi
if i remember correctly,
I didnt get the mean loss so that's the total loss over all pixels * batch
size.

to get the normal loss you can divide the number by pixel number in the
image * batch size.

**Comment by gzYou** [comment]
Thx, I'll try it.

**Comment by gzYou** [comment]
Sorry, another question. I found there are three labels file in the Test_Set. They are "LABELS_obs1","LABELS_obs2_v1" and "LABELS_obs2_v2". So what's the meaning of the file name and which is the true label?

**Comment by Kulbear** [comment]
@gzYou There are multiple (4, I think) annotators and they may annotate the frame multiple times.

The ground truth can be the average across all the annotators.
